<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClients</class>
 <widget class="QDialog" name="DialogClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>369</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>369</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Существующие Еноты</string>
  </property>
  <widget class="QTableView" name="TableViewClients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>711</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">.header {font: 75 8pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAddNewClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Добавить нового клиента</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(227, 227, 227);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDeleteSelectedClient">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Удалить выбранного клиента</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(227, 227, 227);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnEditSelectedClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Редактировать выбранного клиента</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(227, 227, 227);
</string>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(175, 175, 175);
</string>
   </property>
   <property name="title">
    <string>Фильтры</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QComboBox" name="comboBoxFilterClient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFilteringClient">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
